src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -186,6 +186,9 @@ private RowMapBuffer getTransactionRows() throws Exception {
 				case XID:
 					buffer.setXid(event.xidData().getXid());
 
+					// feed metric gauge.
+					replicationLag = event.getEvent().getHeader().getTimestamp() - (System.currentTimeMillis() / 1000);
```

Comment:
The lag is normally quite low. Can we use milliseconds here? So we don't need to `/ 1000`. Otherwise we may find the lag is 0 most of time.